extension/app/ts/components/pages/PersonalSign.tsx

```diff
@@ -178,8 +169,8 @@ export function PersonalSign() {
 								</p>
 							</header>
 							<div class = 'card-content'>
-								<div class = 'control'>
-									<textarea class = 'textarea' readonly ref = { textareaRef } style = 'overflow: hidden; resize: none;'>{ signRequest.message }</textarea>
+								<div class = 'textbox'>
+									<p class = 'paragraph' style = 'color: var(--subtitle-text-color)'>{ signRequest.message }</p>
```

Comment:
Do we render this at all, or is it just a raw byte array?  If it is rendered text, we shouldn't use `break-all`.